src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -949,3 +949,25 @@ def _match_host_names_from_cert(hostnames_from_cert, hostnames_in_webapp):
         elif hostname in hostnames_in_webapp:
             matched.add(hostname)
     return matched
+
+
+def create_function(resource_group_name, name, storage_account, plan=None):
```

Comment:
I believe you need to expose a flag called `consumption-plan`, or you plan to do it later?